challenge_03_loops-conditionals/03_JavaScript_foundations/loops-conditionals/while/exercise_01.js

```diff
@@ -0,0 +1,36 @@
+/*
+    1. Create a while loop that will print out all the even numbers between 10 and 40.
+
+    2. Create a while loop that will print out all the odd numbers between 10 and 40.
+*/
+
+print("Select an option: ");
```

Comment:
This is the output of your script...
```
ReferenceError: print is not defined
```

...have you tested it before pushing? 🤔

⚠️ FIX: spot the 🐛 and fix it